### DISCUSSION TITLE
Up to date hardware list for 2 axis

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#130] [Q&A] [answered]
Ive looked at the hardware list for the 4 Axis version of the controller and there is a hardware list for driving the leadscrew but I also plan on doing the cross slide.

Ive seen some 3d Printed brackets for that, and I imagine I need a similar stepper driver and motor too? Maybe a slightly smaller motor?

Also I cant seem to find any reference to the bracket or any gears that I may need for the cross slide.

Cheers.

**Comment by kachurovskiy** [comment, marked as answer]
@Festivejelly everyone has a different lathe so it's tough to make a hardware list that works for everyone. Even WM210 models vary quite a bit.

For every axis, you need a stepper motor with a driver.

For the main lead screw, some of the motor mount options are listed on https://github.com/kachurovskiy/nanoels/blob/main/hardware.md#stepper-mount

For the cross-slide, see e.g. https://www.youtube.com/watch?v=i7XNa8Kxmfk that shows https://www.thingiverse.com/thing:6058899 calling for a NEMA17 60mm long open loop motor.

**Reply by Festivejelly** [reply]
This is helpful thanks. The belt that connects the leadscrew to the stepper, is that 15mm wide?

I think this is the one? https://www.aliexpress.com/item/1005006095409431.html?spm=a2g0o.productlist.main.17.53f1MO2QMO2Q5y&algo_pvid=ac1a8df9-d5a3-4c44-800a-9228ccd41225&algo_exp_id=ac1a8df9-d5a3-4c44-800a-9228ccd41225-8&pdp_npi=4%40dis%21GBP%219.25%214.62%21%21%2180.00%21%21%40211b81a316972335482792040eec0d%2112000035716182160%21sea%21GB%212408567052%21&curPageLogUid=9VQFPqNwoBw7

**Comment by Nitrous2000** [comment]
this may be a silly question.... but, for simple thread cutting on a chinese lathe, will the single axis controller be sufficient?

**Reply by kachurovskiy** [reply]
Yes, it would be sufficient but it has some limitations - no microstepping on the motor, display doesn't update when motor moves, harder to enter pitch values.

**Comment by Nitrous2000** [comment]
So, would the basic system for cutting a screw be the 4 axis system?

**Reply by kachurovskiy** [reply]
Yes, the recommended controller is the 4-axis one. It's okay to just use 1 axis initially.
